src/views/Room/RollOptionsMenu/index.jsx

```
@@ -0,0 +1,66 @@
+      >
+        <ArrowDropUp />
+      </Button>
+      <Popper
+        sx={{ zIndex: 1 }}
+        open={open}
+        anchorEl={anchorRef.current}
+        role={undefined}
+        transition
+        disablePortal
+        placement="top-end"
+      >
```

Biome

[error] Enforce that elements with ARIA roles must use a valid, non-abstract ARIA role.

Check WAI-ARIA for valid roles or provide options accordingly.
Unsafe fix: Remove the invalid role attribute.
 Check the list of all valid role attributes.


(lint/a11y/useValidAriaRole)